### DISCUSSION TITLE
I have trouble with claiming the Student Pack! Need Help

### DISCUSSION THREAD

**Discussion by AhmedAredah** [community/community#121690] [GitHub Education]
### Hi everyone,

My student ID does not have any date on it. I tried submitting my unofficial transcript, current semester registration, and my I20 (since I am an international student to the US). But all of them did not go through. Any tips on how to proceed?

Thanks.

**Comment by gurjeetsinghvirdee** [comment]
Hey @AhmedAredah 

please go through with this #111352, I'm sure you will find your answer there

**Comment by AhmedAredah** [comment]
@gurjeetsinghvirdee Thanks, I am getting this error now "Your uploaded proof has already been used previously." and this is not mentioned anywhere on that form. thanks for helping

**Reply by gurjeetsinghvirdee** [reply]
In that case use something different proof, like marksheets or you can ask your university to give you bonafide certificate

**Comment by AhmedAredah** [comment]
The proof that I used is the I20. which is an official document stating when I will be finishing my degree. What else can I use? I am out of proofs now tbh.

**Reply by gurjeetsinghvirdee** [reply]
The last option is to ask the university whether they can provide something else. Alternatively, you can ask your friend to apply for this pack if they don't want it or have no use for the GitHub pack.
